web/src/app/schedules/ScheduleOverrideDialog.js

```diff
@@ -35,7 +32,7 @@ const useStyles = makeStyles({
     marginTop: '.3rem',
   },
 })
-export default function ScheduleCalendarOverrideDialog(props) {
+export default function ScheduleOverrideDialog(props) {
```

Comment:
The proptypes on lines 149 and 153 need to be renamed to `ScheduleOverrideDialog.propTypes` otherwise they'll fail on `make check`.

Reply:
Thank you!